Documentation/Upgrade/rook-upgrade.md

```diff
@@ -49,16 +50,18 @@ those releases.
 ## Breaking changes in v1.13
 
 * The minimum supported version of Kubernetes is v1.23.
-* Support for the admission controller/webhooks has been removed. If admission controller/webhooks is enabled, disable by changing
-`ROOK_DISABLE_ADMISSION_CONTROLLER: "true"` in operator.yaml before upgrading to rook v1.13. CRD validation is now enabled with [Common Expression Language](https://kubernetes.io/docs/reference/using-api/cel/). This requires Kubernetes version 1.25 or higher.
-
-## Breaking changes in v1.12
-
-* The minimum supported version of Kubernetes is v1.22.
+  Upgrade to Kubernetes v1.23 or higher before upgrading Rook.
+* The minimum supported version of Ceph is v17.2.0.
+  If a lower version is currently deployed, [Upgrade Ceph](./ceph-upgrade.md) before upgrading Rook.
 * CephCSI CephFS driver introduced a breaking change in v3.9.0. If any existing CephFS storageclass in
```

Comment:
I'd suggest we can remove this csi issue since it would have been handled during the v1.12 upgrade, right?

Reply:
Done